common/app/common/utils/PriceFormatter.java

```diff
@@ -1,10 +1,22 @@
 package common.utils;
 
-import common.contexts.AppContext;
-
 import javax.money.MonetaryAmount;
+import javax.money.format.MonetaryFormats;
+import java.util.Locale;
+
+public class PriceFormatter {
```

Comment:
Remember to document with Javadoc all classes and methods that are not from Pages (those I think they are very self-explained).